test/test_retriever.py

```diff
@@ -19,6 +20,29 @@
                                        shortname='test')
 test_engine.opts = {'database_name': '{db}_abc'}
 HOMEDIR = os.path.expanduser('~')
+test_dir = os.path.dirname(os.path.abspath(__file__))
+
+
+def create_file(data, output='output_file'):
+    """Takes a string and creates a files for tests"""
```

Comment:
"Takes a string and creates a files for tests" -> "Writes a string to a file for use by tests"